.storybook/package.json

```diff
@@ -1 +1,5 @@
-{}
\ No newline at end of file
+{
+  "dependencies": {
+    "webpack": "^5.68.0"
+  }
+}
```

Comment:
@swissspidy do you think we need to import all of webpack here? I added this so that we could add the [env variables as globals in the storybook config](https://github.com/GoogleForCreators/web-stories-wp/pull/5792/files#diff-b185a1de343f74ed35c31ccba4fa43156a89611a58a180bcdc8de9646284f053R90-R104)

Reply:
yeah I think it'd be better empty. Lint was flagging it, but we could probably attack that from our `.eslintrc`?

Reply:
[4e66c25](https://github.com/GoogleForCreators/web-stories-wp/pull/5792/commits/4e66c25dc5fce27a59f392341f332b4c9d71b692) updated that